core/cas-server-core-services/src/main/java/org/apereo/cas/services/DefaultRegisteredServiceAccessStrategy.java

```diff
@@ -412,4 +384,13 @@ public String toString() {
                 .append("rejectedAttributes", this.rejectedAttributes)
                 .toString();
     }
+
+    private static Set<String> objectToSetOfStrings(final Object objVal) {
+        if (objVal instanceof Collection) {
+            final Collection collection = (Collection) objVal;
+            return new HashSet<>(collection);
+        } else {
+            return Collections.singleton(objVal.toString());
```

Comment:
Also not a big deal, but the else clause is rather redundant.  See https://apereo.github.io/cas/developer/Code-Conventions.html#needless-else-clauses